test/testcne/meson.build

```diff
@@ -48,6 +48,7 @@ sources = files(
     'ring_api.c',
     'ring_profile.c',
     'ring_test.c',
+    'tailqs_test.c',
```

Comment:
Can you also update the "test_names" structure below so this test gets run by the CI automatically?